### PR TITLE
Enable Configuration on Install

### PR DESCRIPTION
**Pull request by riptidewave93**
This PR adds support for postinst, which should configure the service & cronjobs to start on next boot of the device.

### CODE REVIEW

File: postinst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Configure our cron
+if [ -d "/etc/crontabs" ]; then
+  echo "PATH=/usr/sbin:/usr/bin:/sbin:/bin" > wrtbwmon
+  echo "* * * * * /usr/sbin/wrtbwmon.sh update /tmp/usage.db" >> wrtbwmon
+  echo "0 * * * * /usr/sbin/wrtbwmon.sh publish /tmp/usage.db /tmp/usage.htm" >> wrtbwmon
+fi
+/etc/init.d/cron reload
+
+# Symlink to our webdir
+if [ -d "/var/www/html" ]; then
```
Comment (pyrovski) [inline postinst]: Should this be "/var/www"?

File: postinst [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# Enable on boot
+/etc/init.d/wrtbwmon enable
+
+# Configure our cron
+if [ -d "/etc/crontabs" ]; then
+  echo "PATH=/usr/sbin:/usr/bin:/sbin:/bin" > wrtbwmon
```
Comment (pyrovski) [inline postinst]: I'm not familiar with postinst scrips. Where does "wrtbwmon" get created? Is it a cron script?

**Review by pyrovski — CHANGES_REQUESTED** [review]
Thanks for fixing this! I had a couple of comments.

File: postinst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  echo "* * * * * /usr/sbin/wrtbwmon.sh update /tmp/usage.db" >> wrtbwmon
+  echo "0 * * * * /usr/sbin/wrtbwmon.sh publish /tmp/usage.db /tmp/usage.htm" >> wrtbwmon
+fi
+/etc/init.d/cron reload
+
+# Symlink to our webdir
+if [ -d "/var/www/html" ]; then
+  ln -s /tmp/usage.htm /www/usage.htm
```
Comment (pyrovski) [inline postinst]: Should this be "/var/www/html/"?

Comment (riptidewave93) [reply postinst]: Yep, that's a typo. Will get that fixed.

File: postinst (hunk 2 above)
Comment (riptidewave93) [reply postinst]: So the concept is that wrtbwmon will be in /etc/crontabs to run as a cron service for wrtbwmon. Looks like I forgot to define the dir, so will fix this as well.

**Comment by riptidewave93** [comment]
Requested changes have been applied.

**Pull request merged**